docker/mock-proxy/nginx-default.conf

```diff
@@ -0,0 +1,11 @@
+server {
+  listen 80;
+  listen 8888;
+
+  location / {
+    proxy_pass http://localhost:17937;
```

Comment:
Using `17937` because the default httpserver port `8080` might be too common and causing issues when running in devnet env. The VM images communicate with BuilderHub through their on-board cvm-proxy instance at port `7937`, and this one is just prefixing it with `1`. See also https://buildernet.org/docs/network-ports